SIG.md

```diff
@@ -0,0 +1,107 @@
+
+### SIG Charter Guide
+All Cilium SIGs must define a charter defining the scope and governance of the SIG.
+The scope must define what areas the SIG is responsible for directing and maintaining.
+The governance must outline the responsibilities within the SIG as well as the roles owning those responsibilities.
+
+### Steps to create a SIG charter
+1. Copy the template into a new file under community/sig-YOURSIG/charter.md
+1. Read the recommendations and requirements so you have context for the template
+1. Fill out the template for your SIG
+1. Update sigs.yaml with the individuals holding the roles as defined in the template.
+1. Create a pull request with a draft of your charter.md and sigs.yaml changes. Communicate it within your SIG and get feedback as needed.
+1. Send the SIG Charter out for review to the Cilium committers.
+1. Once accepted, the committers will ratify the PR by merging it.
+
+### Steps to update an existing SIG charter
+For significant changes, or any changes that could impact other SIGs, such as the scope, create a PR and send it to the committers for review with the subject: "SIG Charter Update: YOURSIG".
+For minor updates to that only impact issues or areas within the scope of the SIG the SIG Chairs should facilitate the change.
+
+### SIG Charter approval process
+When introducing a SIG charter or modification of a charter the following process should be used.
+
+1. Identify a small set of people from the SIG to drive the changes. Most typically this will be the SIG chairs.
+1. Work with the rest of the SIG in question to craft the changes. Make sure to keep the SIG in the loop as discussions progress with the committers.
+1. Work with the committers to gain approval. This can simply be submitting a PR and sending a slack message. If more substantial changes are desired it is advisable to socialize those before drafting a PR.
+1. The committers will be looking to ensure the scope of the SIG as represented in the charter is reasonable (and within the scope of Cilium) and that processes are fair.
+
+# SIG Governance
+
+To standardize Special Interest Group efforts, create maximum transparency, and route contributors to the appropriate SIG; SIGs should follow these guidelines:
+
+- Have an approved Charter.
+- Keep up-to-date meeting notes, linked from the SIG's page in the community repo.
+- Ensure related work happens in a project-owned GitHub org and repository.
+- Use the forums provided as the primary means of working, communicating, and collaborating, as opposed to private emails and meetings.
+- Track and identify all CFPs and other project enhancements related to the SIG area.
+
+## SIG Retirement
+In the event that the SIG is unable to regularly establish consistent quorum or otherwise fulfill its Organizational Management responsibilities
```

Comment:
Do we have a definition of "quorum" or "Organizational Managament" for a SIG? I would keep this as simple as we can, something like "In the event that a SIG is inactive for 3 or more months, it should be retired"